client/layout/guided-tours/button-labels.jsx

```diff
@@ -4,6 +4,7 @@ import { useTranslate } from 'i18n-calypso';
 // Returns React component with a localized label and optional icon
 function button( label, icon ) {
 	return () => {
+		// eslint-disable-next-line wpcalypso/i18n-translate-identifier
 		const tr = useTranslate();
```

Comment:
Wow, what a mess!

@yuliyan look, variable arguments passed to `translate()` 😅

Reply:
I think you got the wrong Julian. :P ping @deBhal

Reply:
I don't think it would cause any problems, but it's indeed a pretty messy solution 😬. I'd personally just pass the translated label as prop, rather than using this component factory pattern, but I assume the original author was looking to avoid component composition for some reason.

Reply:
I definitely didn't ping the wrong one FWIW. Intended to ping this one, from the Global team 😅

Reply:
Ooops, you're right. :) I'm sure the other Julian won't mind, haha!